language-support/ts/codegen/tests/BUILD.bazel

```diff
@@ -101,9 +101,6 @@ sh_test(
             "ts/**/lib/**",
         ],
     ),
-    tags = [
```

Comment:
let’s see how it goes but we might end up having to add this back. You are starting two JVM process + one node process in those tests + damlc so they are very resource hungry and running them in parallel with other tests might end up overloading the machine.

Reply:
Copy that.